src/app/component/MyAccount/MyAccount.component.js

```diff
@@ -115,6 +144,12 @@ class MyAccount extends Component {
               onMouseLeave={ () => this.setState({ isHovered: false }) }
             >
                 { this.renderLoader() }
+                { notification
```

Comment:
Not like this. Maybe we can use the notification component?

Reply:
True. Just found it this morning.